nlp/text_preprocessing.py

```diff
@@ -160,66 +174,578 @@ def numbers_to_words_func(text):
-                     "nimi", "ním", "ňom", "ňou", "ňu", "o", "od", "odo", "on", "ona", "oni", "ono", "ony", "oň",
-                     "oňho", "po", "pod", "podo", "podľa", "pokiaľ", "potom", "popod", "popri", "poza", "práve", "pre",
-                     "prečo", "preto", "pretože", "pred", "predo", "pri", "s", "sa", "si", "sme", "so", "som", "ste",
-                     "sú", "svoj", "svoja", "svoje", "svojho", "svojich", "svojim", "svojím", "svojimi", "svojou",
-                     "svoju", "ta", "tá", "tam", "tak", "takže", "táto", "teda", "tej", "ten", "tento", "tiež", "tí",
-                     "tie", "tieto", "títo", "to", "toho", "tohto", "tom", "tomto", "tomu", "tomuto", "toto", "tou",
-                     "touto", "tu", "tú", "túto", "tvoj", "tvoja", "tvoje", "tvojej", "tvojho", "tvoji", "tvojich",
-                     "tvojím", "tvojimi", "ty", "tých", "tým", "tými", "týmto", "už", "v", "vám", "vami", "vás", "váš",
-                     "vaša", "vaše", "vašej", "vášho", "vaši", "vašich", "vašim", "vaším", "viac", "vo", "však",
-                     "všetci", "všetka", "všetko", "všetky", "všetok", "vy", "z", "za", "začo", "začože", "zo", "že"]
-    stop_words_cz = ["a", "aby", "aj", "ale", "anebo", "ani", "aniz", "ano", "asi", "avska", "az", "ba", "bez", "bude",
-                     "budem", "budes", "by", "byl", "byla", "byli", "bylo", "byt", "ci", "clanek", "clanku", "clanky",
-                     "co", "com", "coz", "cz", "dalsi", "design", "dnes", "do", "email", "ho", "i", "jak", "jake",
-                     "jako", "je", "jeho", "jej", "jeji", "jejich", "jen", "jeste", "jenz", "ji", "jine", "jiz", "jsem",
-                     "jses", "jsi", "jsme", "jsou", "jste", "k", "kam", "kde", "kdo", "kdyz", "ke", "ktera", "ktere",
-                     "kteri", "kterou", "ktery", "ku", "ma", "mate", "me", "mezi", "mi", "mit", "mne", "mnou", "muj",
-                     "muze", "my", "na", "nad", "nam", "napiste", "nas", "nasi", "ne", "nebo", "nebot", "necht",
-                     "nejsou", "není", "neni", "net", "nez", "ni", "nic", "nove", "novy", "nybrz", "o", "od", "ode",
-                     "on", "org", "pak", "po", "pod", "podle", "pokud", "pouze", "prave", "pred", "pres", "pri", "pro",
-                     "proc", "proto", "protoze", "prvni", "pta", "re", "s", "se", "si", "sice", "spol", "strana", "sve",
-                     "svuj", "svych", "svym", "svymi", "ta", "tak", "take", "takze", "tamhle", "tato", "tedy", "tema",
-                     "te", "ten", "tedy", "tento", "teto", "tim", "timto", "tipy", "to", "tohle", "toho", "tohoto",
-                     "tom", "tomto", "tomuto", "totiz", "tu", "tudiz", "tuto", "tvuj", "ty", "tyto", "u", "uz", "v",
-                     "vam", "vas", "vas", "vase", "ve", "vedle", "vice", "vsak", "vsechen", "vy", "vzdyt", "z", "za",
-                     "zda", "zde", "ze", "zpet", "zpravy"]
+    stop_words_sk = [
```

Comment:
Not sure whether this should not be ignored by the linter and be one long line rather than N lines.

Reply:
Maybe we should move this to `nlp/constants` as well.

Reply:
I agree.